resources/index.html

```diff
@@ -84,7 +84,7 @@
 		* [A Comparison of Modern Graphics APIs](https://alain.xyz/blog/comparison-of-modern-graphics-apis)
 		* [Scratchapixel](https://www.scratchapixel.com)
 		* [The Graphics Codex](https://graphicscodex.com/)
-		* [iquilezles.org](https://iquilezles.org/)
+		* [Inigo Quilez's articles and tutorials (and more)](https://iquilezles.org/)
```

Comment:
Hmm why not link to specific articles of his, in various sections... [Link-to-Link](link) is a little thin

Reply:
That's a lot of sections :sweat_smile: But if you'd prefer that I'll make the changes.

Reply:
I mean i understand why you just linked iquilezles as a whole :) if you are on our discord, we could discuss this thing on how to tackle those kind of resources